GjallarhornTest2/Program.fs

```diff
@@ -38,59 +37,91 @@ module Program =
            "Pollux"
            "Avior"
         ]
+    
+    type ItemMessage = UpdateItem
+
+    [<CLIMutable>]
+    type Item =
+        {
+            Name : string
+            Top : float
+            Left : float
+        }
+
+    let createItem name = { Name = name; Top = 200.; Left = 200. }
+
+    [<CLIMutable>]
+    type ItemViewModel =
+        {
+            XName : string
+            Top : float
+            Left : float
+            Self : ISignal<Item>
+        }
 
-    type Msg =
+    let designSignal = createItem "" |> Signal.constant
+    let itemd = { XName = ""; Top = 0.; Left = 0.; Self = designSignal }
+
+    let itemComponent =
```

Comment:
From the CollectionManager example, I seem to understand I need something like this, but I can't figure out where and how I'd need to use it.

Reply:
I can't run this (Your NightSky project isn't there... so the branch fails loading the xaml), but...

A component for the Item would allow you to create messages from the POV of the item itself, and propogate that up to the application. The collection example does this.

Reply:
Hmmm; NightSky should be downloaded via Paket. Does it work if you run the build script?

As for the component, yes, that is what I want, but I don't understand how to use it, as my structure is a little different from the CollectionManager example because I don't have the navigation between different "screens".